dirs/dirs.go

```diff
@@ -25,17 +25,18 @@ import "path/filepath"
 var (
 	GlobalRootDir string
 
-	SnapAppsDir      string
-	SnapOemDir       string
-	SnapDataDir      string
-	SnapDataHomeGlob string
-	SnapAppArmorDir  string
-	SnapSeccompDir   string
-	SnapUdevRulesDir string
-	LocaleDir        string
-	SnapIconsDir     string
-	SnapMetaDir      string
-	SnapLockFile     string
+	SnapAppsDir               string
+	SnapOemDir                string
+	SnapDataDir               string
+	SnapDataHomeGlob          string
+	SnapAppArmorDir           string
+	SnapAppArmorAdditionalDir string
```

Comment:
What's the reason why the additional dir exists? Can we name the field after that reason?